### PR TITLE
Change!

### PR DESCRIPTION
**Pull request by Matanlb**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Matanlb/TestReviewTool/pull/1" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Head line 1!
+## Head line @1@!
```
Comment (Matanlb) [inline README.md]: Do not use 'at' they are evil!!

!fix

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 
-### Some Subject
+### __Some Subject__
```
Comment (Matanlb) [inline README.md]: This is a head line, bold is redandent...

!fix

Comment (Matanlb) [inline README.md]: Done !fixed

File: README.md (hunk 1 above)
Comment (Matanlb) [inline README.md]: Why not? 'at's' are cool!

!

Comment (Matanlb) [inline README.md]: Ok fine... !resolved

Comment (Matanlb) [inline README.md]: !fixed

Comment (Matanlb) [inline README.md]: !fix

Comment (Matanlb) [inline README.md]: !fixed

**Pull request merged**